flake8_class_attributes_order/node_type_weights.py

```diff
@@ -0,0 +1,139 @@
+import warnings
+
+from typing import List, Mapping, Dict
+from typing_extensions import Final
+
+
+non_strict_node_type_weights: Final[Mapping[str, int]] = {
+    'docstring': 0,
+    'pass': 1,
+    'meta_class': 2,
+    'nested_class': 3,
+
+    'constant': 4,
+    'field': 5,
+    'outer_field': 6,
+
+    'if': 7,
+    'expression': 8,
+
+    '__new__': 9,
+    '__init__': 10,
+    '__post_init__': 11,
+    '__str__': 12,
+
+    'save': 13,
+    'delete': 14,
+
+    'property_method': 20,
+    'private_property_method': 20,
+    'static_method': 22,
+    'private_static_method': 22,
+    'class_method': 24,
+    'private_class_method': 24,
+    'method': 26,
+    'magic_method': 27,
+    'private_method': 27,
+}
+
+strict_node_type_weights: Final[Mapping[str, int]] = {
+    'docstring': 0,
+    'pass': 1,
+    'meta_class': 2,
+    'nested_class': 3,
+
+    'constant': 4,
+    'field': 5,
+    'outer_field': 6,
+
+    'if': 7,
+    'expression': 8,
+
+    '__new__': 9,
+    '__init__': 10,
+    '__post_init__': 11,
+    '__str__': 12,
+
+    'save': 13,
+    'delete': 14,
+
+    'property_method': 20,
+    'private_property_method': 21,
+    'static_method': 22,
+    'private_static_method': 23,
+    'class_method': 24,
+    'private_class_method': 25,
+    'method': 26,
+    'magic_method': 27,
+    'private_method': 28,
+}
+
+fixed_node_type_weights: Final[Dict[str, int]] = {
```

Comment:
and this and other global vars inside this module

Reply:
All done!